.env

```diff
@@ -22,4 +22,12 @@ REACT_APP_SWR_LOG_CACHE=false
 REACT_APP_SWR_LOG_REQUESTS=false
 
 # This is the title displayed while the UI is initially loading in the browser
-REACT_APP_DEFAULT_PAGE_TITLE=Estuary Flow
\ No newline at end of file
+REACT_APP_DEFAULT_PAGE_TITLE=Estuary Flow
+
+# LogRocket settings
+REACT_APP_LOGROCKET_APP_ID=tue20y/estuarytest
```

Comment:
This is still my personal test account. Once we see how this works with more than just a local we can make a decision on licensing.